app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,74 @@
+<section class="container flex food-placeholder">
+  <div class="food flex">
+    <h2 class="title"><%= @recipe.name %></h2>
+
+    <div class="flex recipe-detail">
+      <div class="flex detail-col">
+        <div class="flex time">
+          <p>Preparation time: </p>
+          <span><%=@recipe.preparation_time %></span>
+        </div>
+
+        <div>
+          <% if @recipe.public == false %>
+            <%= check_box :public, checked: false  %>
+            <label class="check-label" for="unchecked">Public</label>
+          <% else %>
+            <%= check_box :public, checked: true %>
+            <label class="check-label" for="checked">Public</label>
+          <%end%>
+        </div>
+      </div>
+
+      <div class="flex time">
+        <p>Cooking time: </p>
+        <span><%=@recipe.cooking_time %></span>
+      </div>
+
+      <div>
+        <p><%= @recipe.description %></p>
+      </div>
+
+      <div class="flex detail-col">
+        <div>
+          <%= link_to recipes_path do %>
+            <button>Generate shopping list</button>
+          <% end %>
+        </div>
+        <div>
+          <%= link_to new_recipe_recipe_food_path(@recipe) do %>
+            <button>Add Ingredients</button>
```

Comment:
Recipe show page should allow the user to add an ingredient while clicking on `Add ingredients` button. But there is an error when user clicks on this button. Kindly see the screenshot for error details.

![add_ing_er](https://user-images.githubusercontent.com/35504149/208002739-cf78e4b8-6d52-4783-802a-e5fafd3b64a9.png)

Reply:
We appreciate the review [Sajjad](@SajjadAhmad14), In as much as we added the Add ingredient button, it wasn't part of the requirement for two people. We will disable it to make the work look good.